### PR TITLE
Fix SD-JWT examples; add detail to the 'Securing Verifiable Credentials' section

### PR DESCRIPTION
**Pull request by decentralgabe**
Partially addresses #296 

The next part is to update the respec-vc plugin to generate SD-JWT examples without KB-JWTs with a trailing `~` (https://github.com/w3c/respec-vc/issues/48).

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/pull/304.html" title="Last updated on Sep 26, 2024, 7:17 PM UTC (6741e41)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/304/1a8a350...6741e41.html" title="Last updated on Sep 26, 2024, 7:17 PM UTC (6741e41)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1101,12 +1101,45 @@ <h2 id="conformance-classes">Conformance Classes</h2>
...
+        <p>
+         In all cases, the underlying data model of the Verifiable Credential
+         or Presentation remains consistent with the [[VC-DATA-MODEL-2.0]],
+         but the encoding and security mechanism differ.</p>
+        <p>
+        The normative statements in <a data-cite="VC-DATA-MODEL-2.0#securing-mechanisms">
+        Securing Mechanisms</a> apply to securing
             <code>application/vc-ld+jwt</code> and
             <code>application/vp-ld+jwt</code>,
             <code>application/vc-ld+sd-jwt</code> and
```
Comment (TallTed) [inline index.html:1145]: The examples now use `application/vc-ld+sd` and `application/vp-ld+sd`. These should also be what is in the document, including the lines following the selection above, which are out of range of selection.
```suggestion
            <code>application/vc+jwt</code> and
            <code>application/vp+jwt</code>,
            <code>application/vc+sd-jwt</code> and
```
plus
```
            <code>application/vp+sd-jwt</code>,
            as well as
            <code>application/vc+cose</code> and
            <code>application/vp+cose</code>.
```

Comment (decentralgabe) [reply index.html]: need to revert this. the media types in main still use `-ld` https://w3c.github.io/vc-jose-cose/#media-types

this should be resolved after TPAC and #282

**Review by TallTed — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -237,58 +237,124 @@ <h2 id="conformance-classes">Conformance Classes</h2>
...
-        <p>
-            The JWT Claim Names <code>vc</code> and <code>vp</code>
-            MUST NOT be present in any JWT Claims Set that comprises a
-            Verifiable Credential or a Verifiable Presentation.
-        </p>
-    </section>
+      <section>
+          <h2 id="securing-verifiable-credentials">Securing Verifiable Credentials</h2>
+          <p>
+              The <a data-cite="VC-DATA-MODEL-2.0#securing-mechanism-specifications"></a> describe
```
Comment (selfissued) [inline index.html:243]:
```suggestion
              The <a data-cite="VC-DATA-MODEL-2.0#securing-mechanism-specifications"></a> describes
```

File: index.html [hunk 3]
```diff
@@ -237,58 +237,124 @@ <h2 id="conformance-classes">Conformance Classes</h2>
...
+                  [[VC-DATA-MODEL-2.0]] are secured using JSON Web Tokens, the
+                  header parameter setting of <code>"alg": "none"</code> MUST
+                  be used to communicate that a Verifiable Credential or
+                  Verifiable Presentation encoded as a JWT Claims Set has no
+                  integrity protection. When a Verifiable Credential or
+                  Verifiable Presentation encoded as a JWT Claims Set contains
+                  a <code>proof</code> claim, and the JSON Web Token header
+                  contains <code>"alg": "none"</code>, the JWT Claims Set &mdash;
+                  the Verifiable Credential or Verifiable Presentation &mdash;
+                  MUST be considered to have no integrity protection.
```
Comment (selfissued) [inline index.html:315]:
```suggestion
                  integrity protection.
```

File: index.html [hunk 4]
```diff
@@ -237,58 +237,124 @@ <h2 id="conformance-classes">Conformance Classes</h2>
...
+              </p>
+              <p class="advisement">
+                  Verifiable Credentials and Verifiable Presentations are not
+                  required to be secured or integrity protected or to contain a
+                  <code>proof</code> member.
+              </p>
```
Comment (selfissued) [inline index.html:321]:
```suggestion
```

**Review by selfissued — APPROVED** [review]

**Comment by decentralgabe** [comment]
Editorial approval. Open for over a week. Changes accepted. Merging.

File: index.html [hunk 5]
```diff
@@ -237,58 +237,114 @@ <h2 id="conformance-classes">Conformance Classes</h2>
...
+                  Verifiable Presentation encoded as a JWT Claims Set has no
+                  integrity protection.
+              </p>
+              <p>
+                  Issuers, Holders, and Verifiers MUST ignore all JWT Claim Sets that
+                  have no integrity protection.
+              </p>
+              <p>
+                  The JWT Claim Names <code>vc</code> and <code>vp</code>
+                  MUST NOT be present in any JWT Claim Set.
```
Comment (TallTed) [inline index.html:318]: Is this accurate? We're restricting use of JWT Claim Names `vc` and `vp` in *any* JWT Claim Set, even a JWT Claim Set that does not encode a Verifiable Credential or Verifiable Presentation?

**Pull request merged**